lib/logging.js

```diff
@@ -353,7 +353,7 @@
 
                 entryString += "\n";
 
-                this._pushable = this.push(entryString);
+                this._pushable = this.push(entryString, "utf8");
```

Comment:
probably unnecessary, but it seemed like a reasonable thing to do.  Not married to it.